### PR TITLE
Compute proper bucket priorities for flight mode

### LINKED ISSUE #3931: Flight mode buckets are not loading from inside out
At some point, the correct priority computation got lost (probably my bad :see_no_evil:). Line 147 in flight_bucket_picker needs to be fixed for this.

### PR DESCRIPTION
**Pull request by philippotto**
### URL of deployed dev instance (used for testing):
- https://flightmodepriorities.webknossos.xyz

### Steps to test:
- open flight mode for dataset
- throttle internet speed in devtools
- ensure that buckets are somewhat loading inside-out (don't expect it to be super precise as in ortho mode, since the fallback rendering can intervene here)

### Issues:
- fixes #3931 

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

File: frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js [hunk 1]
```diff
@@ -140,11 +142,14 @@ export default function determineBucketsForFlight(
   };
 
   const fallbackBuckets = isFallbackAvailable ? traverseFallbackBBox(getBBox(planeBuckets)) : [];
-  traversedBuckets = traversedBuckets.concat(fallbackBuckets);
+  // Use a constant priority for the fallback buckets which is higher than the highest non-fallback priority
+  const fallbackPriority = iterStep + 1;
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js:146]: Maybe I'm misunderstanding the code, but isn't `halfWidth + halfWidth + xOffset` the highest non-fallback priority (line 86)?

File: frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js [hunk 2]
```diff
@@ -17,9 +17,9 @@ const getBBox = buckets => ({
   cornerMax: aggregatePerDimension(Math.max, buckets),
 });
 
-function createDistinctBucketAdder(buckets: Array<Vector4>) {
+function createDistinctBucketAdder(bucketWithPriorities: Array<[Vector4, number]>) {
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js:20]:
```suggestion
function createDistinctBucketAdder(bucketsWithPriorities: Array<[Vector4, number]>) {
```

File: frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ function createDistinctBucketAdder(buckets: Array<Vector4>) {
 
     if (!lookupY[z]) {
       lookupY[z] = true;
-      buckets.push(bucketPos);
+      bucketWithPriorities.push([bucketPos, priority]);
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js:31]:
```suggestion
      bucketsWithPriorities.push([bucketPos, priority]);
```

**Review by daniel-wer — COMMENTED** [review]
Cool, works as described in my tests :+1: As you wrote, it's not perfect, but much better than before!

During testing I noticed that lots of segmentation buckets are loaded while in flight mode, although they are not displayed - seems like an easy performance win as well :)

**Review by philippotto — COMMENTED** [review]
Thank you, @daniel-wer, for your valuable feedback! I think, correcting the wrong fallback priority further improved the loading behavior :+1:
Also, I adapted the (pre)fetching code for buckets so that the segmentation layer is not loaded in arbitrary mode. Very interesting that we never noticed this before (but maybe flight mode tracing is often done in datasets without segmentation?). Glad that this is fixed now, too :)

**Review by daniel-wer — APPROVED** [review]
Awesome, it's really fast now :tada:

**Pull request merged**